### PR TITLE
JetElements - Elementor AddOn Trace Added

### PR DESCRIPTION
**Pull request by dhavalveera**
### Technology Request

**Examples:**

https://signalscv.com/
https://blog.stuller.com/
https://www.signpost.com/
https://advertise.jerseydigs.com/
https://www.icontact.com/
https://www.securenetsystems.net/

### CODE REVIEW

File: src/technologies/j.json [hunk 1]
```diff
@@ -279,6 +279,23 @@
...
+    "description": "JetElements is an addon for Elementor that adds additional customisation options to the page builder.",
+    "icon": "jet_elements.png",
+    "js": {
+      "jetElements": ""
+    },
+    "pricing": [
+      "mid",
+      "recurring",
+      "onetime"
+    ],
```
Comment (nurbek91) [inline src/technologies/j.json:295]: image
image

File: src/technologies/j.json [hunk 2]
```diff
@@ -279,6 +279,23 @@
     },
     "website": "https://jenkins.io/"
   },
+  "JetElements": {
```
Comment (nurbek91) [inline src/technologies/j.json:282]: Crocoblock JetElements

File: src/technologies/j.json [hunk 3]
```diff
@@ -279,6 +279,23 @@
...
+    "cats": [
+      87
+    ],
+    "description": "JetElements is an addon for Elementor that adds additional customisation options to the page builder.",
+    "icon": "jet_elements.png",
```
Comment (nurbek91) [inline src/technologies/j.json:287]: Crocoblock
svg available

**Review by nurbek91 — CHANGES_REQUESTED** [review]

File: package.json [hunk 4]
```diff
@@ -22,7 +22,7 @@
     "lint:fix": "eslint --fix src/**/*.{js,json}",
     "validate": "yarn run lint && jsonlint -qV ./schema.json ./src/technologies/ && node ./bin/validate.js",
     "convert": "node --no-warnings ./bin/convert.js",
-    "prettify": "jsonlint -si --trim-trailing-commas --enforce-double-quotes ./src/categories.json ./src/technologies/*.json",
+    "prettify": "jsonlint -si --trim-trailing-commas --enforce-double-quotes ./src/categories.json ./src/technologies/m.json",
```
Comment (nurbek91) [inline package.json:25]: is this update needed?

**Review by nurbek91 — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (dhavalveera) [reply package.json:25]: this isn't required, actually in my windows when I run `yarn run prettify` that time this => `"prettify": "jsonlint -si --trim-trailing-commas --enforce-double-quotes ./src/categories.json ./src/technologies/*.json"` command isn't working for me, it's saying no files found so for that I've pass the file name to prettify it

**Review by nurbek91 — APPROVED** [review]

**Pull request merged**